datadog_checks_dev/datadog_checks/dev/tooling/create.py

```diff
@@ -42,7 +43,7 @@ def construct_template_fields(integration_name, repo_choice, **kwargs):
 
 2. Run `ddev release build {normalized_integration_name}` to build the package.
 
-3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+3. [Download the Datadog Agent][2].
```

Comment:
There's still an issue when creating a check with `ddev create Test --type check`- both the agent and autodiscovery are link 2

Reply:
Thanks, good catch!